docs/src/pages/components/image-list/CustomImageList.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import ImageList from '@material-ui/core/ImageList';
+import ImageListItem from '@material-ui/core/ImageListItem';
+import ImageListItemBar from '@material-ui/core/ImageListItemBar';
+import IconButton from '@material-ui/core/IconButton';
+import StarBorderIcon from '@material-ui/icons/StarBorder';
+import itemData from './itemData';
```

Comment:
A couple of thoughts on the data source:

- The file doesn't look worse than the top 100 movies we include in all the autocomplete demos. What about we include it in the demo?
- In the future, we could consolidate on a generic name to load data, and have the codesandbox generation script automatically handle it. For now, I doubt we need to care.
- What about we use a single set of data instead of two?
- What about we pull all the images from Unsplash so we can leverage imgix (what they use) to optimize the speed? e.g. srcset

Reply:
> The file doesn't look worse than the top 100 movies we include in all the autocomplete demos. What about we include it in the demo?

And add 100 images?

Reply:
> What about we use a single set of data instead of two?

Redundant question, but again, autocomplete doesn't have images.

Reply:
> What about we pull all the images from Unsplash 

Possibly

Reply:
> And add 100 images?

I think that the current number of images is great, after, we would need to look into lazy loading or even virtualization.

Reply:
> What about we include it in the demo?

Oh, right, you mean include the current data in the demo, not that from the autocomplete! 😄